### ISSUE TITLE
[Bug]: the mo start failed after crashing down

### ISSUE THREAD

**Issue by aressu1985** [matrixorigin/matrixone#5336]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):0.6.0
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

022/09/28 23:19:26.034759 +0800 INFO fileservice/memory_fs.go:165 error: file cnstore/metadata.data is not found {"span": {"TraceId": "00000000-0000-0000-0000-000000000000", "SpanId": "0000000000000000"}}
2022/09/28 23:19:26.034817 +0800 INFO cnservice/server.go:161 Shutdown The Server With Ctrl+C | Ctrl+\.
2022/09/28 23:19:26.034831 +0800 INFO cnservice/server.go:169 Initialize the engine ...
2022/09/28 23:19:26.038681 +0800 INFO batchstoredriver/file.go:41 error: internal error: parse version error {"span": {"TraceId": "00000000-0000-0000-0000-000000000000", "SpanId": "0000000000000000"}}
2022/09/28 23:19:26.038715 +0800 INFO batchstoredriver/file.go:41 error: internal error: parse version error {"span": {"TraceId": "00000000-0000-0000-0000-000000000000", "SpanId": "0000000000000000"}}
2022/09/28 23:19:26.038730 +0800 INFO batchstoredriver/file.go:41 error: internal error: parse version error {"span": {"TraceId": "00000000-0000-0000-0000-000000000000", "SpanId": "0000000000000000"}}
2022/09/28 23:19:26.042981 +0800 INFO batchstoredriver/file.go:41 error: internal error: parse version error {"span": {"TraceId": "00000000-0000-0000-0000-000000000000", "SpanId": "0000000000000000"}}
2022/09/28 23:19:26.043049 +0800 INFO batchstoredriver/file.go:41 error: internal error: parse version error {"span": {"TraceId": "00000000-0000-0000-0000-000000000000", "SpanId": "0000000000000000"}}
2022/09/28 23:19:26.043067 +0800 INFO batchstoredriver/file.go:41 error: internal error: parse version error {"span": {"TraceId": "00000000-0000-0000-0000-000000000000", "SpanId": "0000000000000000"}}
2022/09/28 23:19:26.043081 +0800 INFO batchstoredriver/file.go:41 error: internal error: parse version error {"span": {"TraceId": "00000000-0000-0000-0000-000000000000", "SpanId": "0000000000000000"}}
2022/09/28 23:19:26.043097 +0800 INFO batchstoredriver/file.go:41 error: internal error: parse version error {"span": {"TraceId": "00000000-0000-0000-0000-000000000000", "SpanId": "0000000000000000"}}
2022/09/28 23:19:26.043122 +0800 INFO batchstoredriver/file.go:41 error: internal error: parse version error {"span": {"TraceId": "00000000-0000-0000-0000-000000000000", "SpanId": "0000000000000000"}}
2022/09/28 23:19:26.043145 +0800 INFO batchstoredriver/file.go:41 error: internal error: parse version error {"span": {"TraceId": "00000000-0000-0000-0000-000000000000", "SpanId": "0000000000000000"}}
2022/09/28 23:19:26.043159 +0800 INFO batchstoredriver/file.go:41 error: internal error: parse version error {"span": {"TraceId":

**Comment by reusee** [comment]
Lack of reproducing steps.

**Comment by aressu1985** [comment]
@reusee assign this issue to @aressu1985 please

**Comment by aressu1985** [comment]
fixed:fc685762b89cd931bdac2c09b93f6bab4ceeb6d5
